ballerina/types.bal

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) 
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Connection options for Redis Client Endpoint.
+#
+# + clientName - The clientName of the connection
+# + connectionPooling - Boolean value depending on whether the connection
+#   pooling is enabled or not
+# + isClusterConnection - Whether to enable cluster connection or not
+# + ssl - Boolean value depending on whether SSL is enabled or not
+# + startTls - Boolean value depending on whether startTLS is enabled or not
+# + verifyPeer - Boolean value depending on whether peer verification is
+#   enabled or not
+# + database - The database to be used with the connection
+# + connectionTimeout - The timeout value for the connection
+public type Options record {|
+    @display{label: "Client Name"} 
+    string clientName = "";
```

Comment:
Can't we make this optional ?

Reply:
Yeah we can do that as its a non-breaking change. Added with https://github.com/ballerina-platform/module-ballerinax-redis/pull/197/commits/6d826c80794537dc41bc07fc1bf660473cc4c8c5